### ISSUE TITLE
How can I run the training code?

### ISSUE THREAD

**Issue by toshinyg** [lhoyer/MIC#28]
Dear LuKas,

First of all, thank you for providing valuable experiment results and code. I attempted to train the classification UDA model, but encountered a "No such file or directory" error.

I followed the installation instructions in the README and executed the command "python run_experiments.py --exp 2". However, I received the following error message after some sentences:

"Run config 1/13: {'exp': 1, 'name': 'office_ar2cl_cdan_mcc_sdat_masking_m64-07-a09-plw-cj02-02-b_vit', 'subfolder': 'examples', 'NGPUS': 1, 'NCPUS': 8, 'gpu_model': 'NVIDIATITANRTX', 'EXEC_CMD': 'python cdan_mcc_sdat_masking.py data/office-home -d OfficeHome -s Ar -t Cl --epochs 30 -b 24 --no-pool --lr 0.002 --seed 0 -a vit_base_patch16_224 --gpu 0 --rho 0.02 --alpha 0.9 --pseudo_label_weight prob --mask_block_size 64 --mask_ratio 0.7 --mask_color_jitter_s 0.2 --mask_color_jitter_p 0.2 --mask_blur True --log logs/cdan_mcc_sdat_vit/office_ar2cl --log_name office_ar2cl_cdan_mcc_sdat_masking_m64-07-a09-plw-cj02-02-b_vit --log_results'}: No such file or directory"

Is there anything else I need to do in order to run the code successfully?

Thank you.
